k8s.gcr.io/manifests/k8s-staging-kubetest2/promoter-manifest.yaml

```diff
@@ -1,50 +0,0 @@
-# google group for gcr.io/k8s-staging-kubetest2 is k8s-infra-staging-kubetest2@kubernetes.io
-registries:
-- name: gcr.io/k8s-staging-kubetest2
-  src: true
-- name: us.gcr.io/k8s-artifacts-prod/kubetest2
-  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
-- name: eu.gcr.io/k8s-artifacts-prod/kubetest2
```

Comment:
Some are not empty, like this one here. Is that expected?

Reply:
There are just references to where the image-promoter is supposed to push the images for promotion. 
As long there is no promotion of the images stored on those staging repos, deleting them should be ok.

Reply:
OK, SGTM